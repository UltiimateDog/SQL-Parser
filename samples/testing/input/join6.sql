SELECT Student.B, Student.A, Enrolled.H FROM Student, Enrolled WHERE Student.A = Enrolled.A AND Student.B > 100;