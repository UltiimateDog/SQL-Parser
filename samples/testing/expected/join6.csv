200,1,75
200,1,82
200,1,92
200,2,12
